src/renderer/reducers/settings.js

```diff
@@ -383,6 +385,8 @@ export const swapProvidersSelector = (state: Object) => state.settings.swapProvi
 
 export const showClearCacheBannerSelector = (state: Object) => state.settings.showClearCacheBanner;
 
+export const fullNodeEnabledSelector = (state: Object) => state.settings.fullNodeEnabled;
```

Comment:
I don't understand why we need to track this boolean and where we would not be able to replace that usage by a `useEnv("SATSTACK")`